internal-packages/ai/src/analysis-plugins/plugins/epistemic-critic/EpistemicIssue.ts

```
@@ -0,0 +1,349 @@
+        }
+        return prefix ? `${prefix} missing context` : "Missing context";
+
+      case ISSUE_TYPES.DECEPTIVE_WORDING:
+        if (adjustedSeverity >= 70) {
+          return "Deceptive presentation";
+        }
+        return prefix ? `${prefix} misleading wording` : "Misleading wording";
+
+      case ISSUE_TYPES.LOGICAL_FALLACY:
+        const fallacyName = this.extractFallacyName();
+        if (fallacyName) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)